Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Remote_PsExec_with_LOLBIN_command_execution_alert.yml

```diff
@@ -0,0 +1,1083 @@
+      version: -1
+      name: Found additional alerts or suspicious command line?
+      description: Check if any malicious verdict is found.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "22"
+      Malicious:
+      - "15"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              complex:
+                root: CommandlineVerdict
+            iscontext: true
+          right:
+            value: {}
+        - operator: isExists
+          left:
+            value:
+              complex:
+                root: PaloAltoNetworksXDR
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: 1c1d53d7-26ca-40a4-8dc7-8e89d6113644
+    type: regular
+    task:
+      id: 1c1d53d7-26ca-40a4-8dc7-8e89d6113644
+      version: -1
+      name: Get alert data
+      description: "Returns a list of alerts and their metadata, which you can filter by built-in arguments or use the custom_filter to input a JSON filter object. \nMultiple filter arguments will be concatenated using the AND operator, while arguments that support a comma-separated list of values will use an OR operator between each value."
+      script: Cortex XDR - IR|||xdr-get-alerts
+      type: regular
+      iscommand: true
+      brand: Cortex XDR - IR
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      alert_id:
+        simple: ${inputs.alerts_ids}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: 1ee53496-9642-4564-86d9-59940ca07a5a
+    type: playbook
+    task:
+      id: 1ee53496-9642-4564-86d9-59940ca07a5a
+      version: -1
+      name: Threat Hunting - Generic
+      description: "This playbook enables threat hunting for IOCs in your enterprise. It currently supports the following integrations: \n- Splunk\n- Qradar\n- Pan-os \n- Cortex data lake \n- Autofocus\n- Microsoft 365 Defender"
+      playbookName: Threat Hunting - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "29"
+    scriptarguments:
+      IPAddress:
+        complex:
+          root: inputs.SrcIPAddress
+          transformers:
+          - operator: uniq
+      MD5:
+        complex:
+          root: File
```

Comment:
For all the hashes in this subplaybook input - we talked about that we do not want all files (as is can be CMD, PowerShell, py, etc). you wanted to hunt the file that was used for the attack.

Reply:
same

Reply:
Also for all the has inputs (different SHAs)